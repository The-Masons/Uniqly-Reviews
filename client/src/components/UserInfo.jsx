import React from 'react';
import ReactDOM from 'react-dom';
import axios from 'axios';

var UserInfo = (props)=>{
    return (
            <div className='review-userinfo'>
                <div className='userinfo-nickname'>{props.reviewData.nickname}</div>
                <div><b>From: </b>{props.reviewData.from}</div>
                <div><b>Size Purchased: </b>{props.reviewData.size_purchased}</div>
                <div><b>Height: </b>{props.reviewData.height}</div>
                <div><b>Age: </b>{props.reviewData.age}</div>
                <div><b>Gender: </b>{props.reviewData.gender}</div>
                <div><b>Chest size: </b>{props.reviewData.chest_size}</div>
                <div><b>Waist size: </b>{props.reviewData.waist_size}</div>
                <div><b>Hips size: </b>{props.reviewData.hip_size}</div>
                <div><b>Waist size: </b>{props.reviewData.waist_size}</div>
            </div>
    )
}

export default UserInfo;
