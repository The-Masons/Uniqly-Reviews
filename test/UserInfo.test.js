import React from 'react';
import ReactDOM from 'react-dom';
import UserInfo from '../client/src/components/UserInfo.jsx';
import renderer from 'react-test-renderer';

const sample = {
  "product_id": 7,
  "nickname": "Vickie",
  "from": "South Jacinthe, Illinois",
  "size_purchased": "XXL",
  "age": "45 to 54",
  "gender": "F",
  "received_free_product": false,
  "chest_size": 50,
  "waist_size": 32,
  "height": "5'6",
  "hips_size": 31,
  "product_fit": 3,
  "product_length": 2,
  "product_quality": 5,
  "date": "2017-06-17T11:27:51.445Z",
  "rating": 2,
  "review_title": "molestias sit est",
  "review_body": "Cumque id repellendus sed eum eaque illum vitae consequatur. Voluptatem et perspiciatis porro omnis iure quos ea. Incidunt blanditiis ut qui vero odio soluta. Suscipit consequatur nesciunt.",
  "recommended": true,
  "helpful": 11,
  "from_uk": true,
  "verified_purchaser": false,
  "employee": true
};

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<UserInfo reviewData={sample}/>, div);
});

describe('UserInfo', () => {
  test('should render with the correct items', () => {
    const itemPicker = mount(
      <UserInfo
        reviewData={sample}/>
      );
    expect(UserInfo).toMatchSnapshot();
  });
});