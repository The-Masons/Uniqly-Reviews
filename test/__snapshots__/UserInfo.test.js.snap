// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserInfo should render with the correct items 1`] = `[Function]`;
